### ISSUE TITLE
Unexisting input file referenced 06-visualization-ggplot-python

### ISSUE THREAD

**Issue by bgbg** [datacarpentry/python-ecology-lesson#208]
The 06-visualization-ggplot-python lesson references surveys_complete.csv. This file is not a part of this repo and is not created in any step of this course. From reading the corresponding R course [here](http://www.datacarpentry.org/R-ecology-lesson/03-dplyr.html), I understand that this file is based on the original survey data with missing values removed. However, after removing the missing values, the following code from the Python lecture doesn't work due to the missing 
`species` column:

```
yearly_counts = surveys_complete[['year','species_id','species']].groupby(['year', 'species_id']).count().reset_index()
yearly_counts.columns = ['year','species_id', 'n']
```

---

Thanks for contributing! If this contribution is for instructor training, please send an email to checkout@carpentries.org with a link to this contribution so we can record your progress. You’ve completed your contribution step for instructor checkout just by submitting this contribution.  

Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution. Although not all contributions can be incorporated into the lesson materials, we appreciate your time and effort to improve the curriculum.  If you have any questions about the lesson maintenance process or would like to volunteer your time as a contribution reviewer, please contact Kate Hertweck (k8hertweck@gmail.com).  

---

**Comment by wrightaprilm** [comment]
OK, this should be fixed as of #commit  dcb23abce. I'm going to leave the issue open to make some flavor text about where this file comes from, or possibly to have it be generated in a previous lesson (feel free to untag yourself).

**Comment by wrightaprilm** [comment]
Alright, I redid some things. Per b247c566, lesson 03 now generates the surveys_complete file. As or 5acd5b, I've refactored the time series example to use a different set of groupby commands to generate the data that were in the "species" column, so we don't have to introduce a separate file at all. In b247c566, I added to the instructor guide the information that there is now a sample_output folder that contains the file made at the end of 03, in case someone had something weird go wrong in a workshop. Then they can use the precooked output. I had a run through of 06, and it looks like it all works.
